tools/lint-docstring

```diff
@@ -157,6 +222,12 @@ def column_text_with_spaces(
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
-        description="Lint and fix docstrings in all ZulipTerminal files"
+        description="Lint and fix docstrings in all zulip-terminal files"
+    )
+    parser.add_argument(
+        "--fix",
+        action="store_true",
+        help="Update developer-file-overview with docstrings of all files",
```

Comment:
Here the description and help text sound opposite to each other. The description sounds like we are linting and fixing the docstrings based on the markdown file but we are doing the opposite.

Reply:
Even the name of the file sounds that way.